src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,40 @@
+import cn from 'classnames';
+import { Filter } from '../../enum/Filter';
+
+type Props = {
+  filter: Filter,
+  setFilter: (value: Filter) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({ filter, setFilter }) => {
+  return (
+    <nav className="filter">
+      <a
+        href="#/"
+        className={cn('filter__link',
```

Comment:
Don't cut variable name
```suggestion
        className={className('filter__link',
```

Reply:
not fixed